### PR TITLE
Added links to JHipster: Build and Deploy Spring Boot Microservices

### PR DESCRIPTION
**Pull request by anataliocs**
Added links to the **MooC** section on the index page for:

_JHipster: Build and Deploy Spring Boot Microservices_
with: Chris Anatalio

**LinkedIn Learning:**
https://www.linkedin.com/learning/jhipster-build-and-deploy-spring-boot-microservices?trk=insiders_43129714_learning

**Lynda.com:**
https://www.lynda.com/course-tutorials/JHipster-Build-Deploy-Microservices/624352-2.html

### CODE REVIEW

**Comment by mraible** [comment]
If both courses are the same, please change to just one link.

**Comment by deepu105** [comment]
Yes please, they are the same so plz add only one link

**Comment by anataliocs** [comment]
They are the same course but hosted on different platforms.

Some people might have a subscription to Lynda.com while others are subscribed to LinkedIn Premium.  There are ways to transfer your membership but it's currently a clunky process and only applicable to certain membership types so the majority of users just stick to their respective platform.

My Content Manager at LinkedIn let me know that users generally stick to 1 platform or the other because they are accustomed to the UI/UX of their respective platform which is why we are currently maintaining both platforms.

But if we really just want to just display one link, then we are losing part of the potential audience, but I'll make that change if we really insist on it.

**Comment by deepu105** [comment]
May be you could put both links on a page in your website and then link to that here. Anyway will let @jdubois make the decision

**Comment by anataliocs** [comment]
Maybe we can do the course name and then 2 sub-bullets like:

- **JHipster: Build and Deploy Spring Boot Microservices by Chris Anatalio**
  - [Available on LinkedIn Learning](https://www.linkedin.com/learning/jhipster-build-and-deploy-spring-boot-microservices?trk=insiders_43129714_learning)
  - [And on Lynda.com](https://www.lynda.com/course-tutorials/JHipster-Build-Deploy-Microservices/624352-2.html)

**Comment by deepu105** [comment]
Can you instead put the name and then in brackets add the links?

**Comment by anataliocs** [comment]
@deepu105 Hey I'm sorry I'm not following.  Can you give me an example of the syntax you want?

**Comment by deepu105** [comment]
JHipster: Build and Deploy Spring Boot Microservices by Chris Anatalio (LinkedIn Learning, Lynda.com)

**Comment by anataliocs** [comment]
@deepu105 Ok updated the format to take up one line as indicated above.

File: index.html [hunk 1]
```diff
@@ -248,6 +248,7 @@ <h3>New online trainings (MOOCs)</h3>
                         <ul>
                             <li><a href="https://therealdanvega.teachable.com/p/jhipster/?product_id=456739&coupon_code=JHIPSTER" target="_blank">Angular 4 Java Developers by Dan Vega and John Thompson</a> (by using this link, you will get a big discount over the normal price!)</li>
                             <li><a href="https://www.pluralsight.com/courses/scaffolding-spring-boot-and-angular-with-jhipster" target="_blank">Scaffolding Spring Boot and Angular Web Apps with JHipster by Michael Hoffman</a></li>
+                            <li>JHipster: Build and Deploy Spring Boot Microservices by Chris Anatalio (<a href="https://www.linkedin.com/learning/jhipster-build-and-deploy-spring-boot-microservices?trk=insiders_43129714_learning" target="_blank">LinkedIn Learning</a> , <a href="https://www.lynda.com/course-tutorials/JHipster-Build-Deploy-Microservices/624352-2.html" target="_blank">Lynda.com</a>)</li>
```
Comment (mraible) [inline index.html]: Remove space before comma in "LinkedIn Learning</a> ,"

**Comment by anataliocs** [comment]
@mraible Good catch!  Thanks.  Updated.

**Comment by mraible** [comment]
LGTM!

**Comment by anataliocs** [comment]
@jdubois and @deepu105 Did you have any additional feedback?

**Comment by jdubois** [comment]
Nope, I'm merging!

**Comment by jdubois** [comment]
Thanks & congrats @anataliocs - you're on the front page :-)

**Pull request merged**